cves/kernel/CVE-2021-33624.yml

```diff
@@ -245,10 +261,11 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 17a5267067f3c372fec9ffb798d6eaba6b5e6a4c
+    note: Interesting because it adds a verifier core which records the states
```

Comment:
More information about the verifier core and how it relates to bug could be useful